### PR TITLE
chore: update release branch

### PR DESCRIPTION
**Pull request by dalelane**
- add mime types for new asyncapi version
- update release branch name in package.json

Contributes to: asyncapi/spec#675

Signed-off-by: Dale Lane <dale.lane@uk.ibm.com>

### CODE REVIEW

**Review by derberg — APPROVED** [review]
LGTM 🚀

@fmvilas have a look

File: lib/asyncapiSchemaFormatParser.js [hunk 1]
```diff
@@ -47,12 +47,15 @@ function getMimeTypes() {
...
     'application/vnd.aai.asyncapi;version=2.2.0',
+    'application/vnd.aai.asyncapi;version=2.3.0',
     'application/vnd.aai.asyncapi+json;version=2.0.0',
     'application/vnd.aai.asyncapi+json;version=2.1.0',
     'application/vnd.aai.asyncapi+json;version=2.2.0',
+    'application/vnd.aai.asyncapi+json;version=2.3.0',
     'application/vnd.aai.asyncapi+yaml;version=2.0.0',
     'application/vnd.aai.asyncapi+yaml;version=2.1.0',
     'application/vnd.aai.asyncapi+yaml;version=2.2.0',
+    'application/vnd.aai.asyncapi+yaml;version=2.3.0',
```
Comment (fmvilas) [inline lib/asyncapiSchemaFormatParser.js:58]: At this point, I'm already wondering if we should refactor this to something like:

```suggestion
    ...['2.0.0', '2.1.0', '2.2.0', '2.3.0'].map((version) => {
      [
        `application/vnd.aai.asyncapi;version=${version}`,
        `application/vnd.aai.asyncapi+json;version=${version}`,
        `application/vnd.aai.asyncapi+yaml;version=${version}`,
      ]
    }).flat()
```

What do you think?

Comment (derberg) [reply lib/asyncapiSchemaFormatParser.js:58]: @fmvilas  Definitely makes sense as then it is just a matter of adding new version to the list once, no copy/paste.

Just small improvement, missing return and generic mime types:
```
function getMimeTypes() {
  return [
    ...['2.0.0', '2.1.0', '2.2.0', '2.3.0'].map((version) => {
      return [
        `application/vnd.aai.asyncapi;version=${version}`,
        `application/vnd.aai.asyncapi+json;version=${version}`,
        `application/vnd.aai.asyncapi+yaml;version=${version}`,
      ]
    }).flat().concat(['application/schema;version=draft-07', 'application/schema+json;version=draft-07', 'application/schema+yaml;version=draft-07'])
  ];
}
```

**Comment by derberg** [comment]
@fmvilas updated, you can approve

@dalelane I'm not sure about your availability now between Christmas and new year so took liberty to improve PR directly 😉

**Review by fmvilas — APPROVED** [review]

**Pull request merged**